src/engine/Service.cpp

```diff
@@ -201,14 +202,15 @@ void Service::writeJsonResult(const std::vector<std::string>& vars,
   IdTableStatic<I> idTable = std::move(*idTablePtr).toStatic<I>();
   checkCancellation();
   std::vector<size_t> numLocalVocabPerColumn(idTable.numColumns());
+  ad_utility::HashMap<std::string, Id> blankNodeMap;
```

Comment:
Add a `TODO<joka921> We should include a memory limit, as soon as we can do proper memory-limited HashMaps`.